services/intersection-api/api/src/main/java/us/dot/its/jpo/ode/api/accessors/events/aggregations/bsmmessagecountprogression/BsmMessageCountProgressionEventAggregationRepository.java

```diff
@@ -0,0 +1,17 @@
+package us.dot.its.jpo.ode.api.accessors.events.aggregations.bsmmessagecountprogression;
```

Comment:
These multi-word all lower-case paths are funky - is this the recommended way to format java paths?